lib/host-sp-messages/src/lib.rs

```diff
@@ -282,6 +291,9 @@ pub enum InventoryData {
         ic_device_id: [u8; 4],
         /// IC_DEVICE_REV, PMBus operation 0xAE
         ic_device_rev: [u8; 4],
+
+        voltage_sensors: [SensorId; 3],
```

Comment:
A general question I have while looking at this, is since it's on me to encode the semantics of each sensor, how do I figure out what this slot is and map it back to the corresponding entry? In this case I assume these are all output values related to the different rails.